test/requirements.in

```diff
@@ -11,7 +11,7 @@ scikit-image
 shap
 pytest
 wandb
-huggingface_hub
+git+https://github.com/huggingface/huggingface_hub.git@57d23eee58af01611ddade8406e08166db3fcfa1
```

Comment:
This worked for me: 

```
huggingface_hub@git+https://github.com/huggingface/huggingface_hub@57d23eee58af01611ddade8406e08166db3fcfa1
```

Following this guide: https://github.com/pypa/hatch/blob/master/docs/config/dependency.md#version-control-systems

Then you have to add this line to `pyproject.toml`

```
[tool.hatch.metadata]
allow-direct-references = true
```

But shouldn't we just wait for the next hub release?

Reply:
I'm in favor of waiting to merge this when `send_telemetry` is released.

Reply:
Word!